packages/meditrak-app-server/src/sync/ServerChangeEnqueuer.ts

```diff
@@ -0,0 +1,64 @@
+/**
+ * Tupaia
+ * Copyright (c) 2017 - 2022 Beyond Essential Systems Pty Ltd
+ */
+
+import { ChangeHandler } from '@tupaia/database';
+import { MeditrakSyncQueueModel } from '../models';
+import { MeditrakAppServerModelRegistry } from '../types';
+
+type Change = { record_id: string; record_type: string; type: 'update' | 'delete' };
+
+/**
+ * Adds server side changes to the meditrakSyncQueue
+ */
+export class ServerChangeEnqueuer extends ChangeHandler {
+  private readonly syncQueueModel: MeditrakSyncQueueModel;
+
+  public constructor(models: MeditrakAppServerModelRegistry) {
+    super(models);
+    this.syncQueueModel = models.meditrakSyncQueue;
+
+    const stripDataFromChange = (
+      change: {
+        old_record: Record<string, unknown>;
+        new_record: Record<string, unknown>;
+      } & Change,
+    ) => {
+      // eslint-disable-next-line @typescript-eslint/naming-convention
+      const { old_record, new_record, ...restOfRecord } = change;
+      return [{ ...restOfRecord }];
+    };
```

Comment:
This data needs to be stripped out, as we just keep track of the record_id for each change